README.md

```diff
@@ -2235,19 +2239,19 @@ Things to look for in
   You know what this is now.
 - Bonus: Additional cash you’ll get on a regular basis, if the company has a plan for this.
 - [Signing bonus](https://www.investopedia.com/terms/s/signing-bonus.asp): Cash you get just for
-  signing. Note signing bonuses usually have some strings attached—for example you will have
-  to pay it back if you leave the company within 12 or 24 months.
+  signing. (Signing bonuses usually have some contingencies—for example, you could have to
```

Comment:
What was wrong with "strings attached"? It is an informal term, but for that reason safer to use when describing things informally.

*Contingency* has other technical meanings so can be confused (for example contingency fees are a different thing), so this change is likely not a "copy-edit-only" as it has some danger.

Reply:
I did not anticipate the term's technical reference. Nothing wrong with "strings attached," just wanted to tighten the sentence. Fixed!